homework29/my-app/src/App.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Component } from 'react';
+import './App.css';
+import Emoji from './Emoji/Emoji';
+
+class App extends Component {
+  constructor() {
+  super();
+    this.state = {
+      isOpen: false,
+      resaltShow: false,
+      emojiWon:'',
+      emojis: [
+        { unicode:'128512', like:0},
+        { unicode: '128105', like:0 },
+        { unicode: '128514', like:0 },
+        { unicode: '127877', like:0},
+        { unicode:'128692', like:0}
+      ],
+      
+    }
+  } 
+
+
+  handlerOnClick (index) {
```

Comment:
```suggestion
  handleClick (index) {
```
так будет правильнее